packages/core/src/abstract-dialect/query-generator-typescript.ts

```diff
@@ -443,16 +455,189 @@ export class AbstractQueryGeneratorTypeScript<Dialect extends AbstractDialect =
     throw new Error(`showConstraintsQuery has not been implemented in ${this.dialect.name}.`);
   }
 
-  showIndexesQuery(_tableName: TableOrModel): string {
-    throw new Error(`showIndexesQuery has not been implemented in ${this.dialect.name}.`);
+  addIndexQuery(tableOrModel: TableOrModel, options: AddIndexQueryOptions): string {
```

Comment:
```suggestion
  addIndexQuery(tableName: TableOrModel, options: AddIndexQueryOptions): string {
```
I have added the suggestion to sqlite as well, but this suggestion holds for all instances where `tableOrModel` is used. `tableName` does not cover all use cases but it's the term we use in other instances

Reply:
Done